scripts/core/ui/ui.js

```diff
@@ -1201,7 +1176,7 @@ export default angular.module('superdesk.core.ui', [
     }])
 
     .directive('sdShadow', ShadowDirective)
-    .directive('sdToggleBox', ToggleBoxDirective)
+    // .directive('sdToggleBox', ToggleBoxDirective)
```

Comment:
Why is this commented out? Just remove it please. We shouldn't commit dead code.

Reply:
true. We are having pressure here before presentation :)